### DISCUSSION TITLE
Incorrect unfocused-split-fill after config reload

### DISCUSSION THREAD

**Discussion by Amparose** [ghostty-org/ghostty#3562] [Q&A] [answered]
Is there a way to change the inactive split decoration?

It's quite ghastly for rose-pine. I couldn't find any options to change the opacity/colour of the overlay.

![CleanShot 2024-12-27 at 20 53 51@2x](https://github.com/user-attachments/assets/31ed972a-8927-4ad0-9bc2-43108626f506)

**Comment by 00-kat** [comment]
The configuration calls it the "unfocused split", which is probably why you didn't find it.  [`unfocused-split-opacity`](https://ghostty.org/docs/config/reference#unfocused-split-opacity) and [`unfocused-split-fill`](https://ghostty.org/docs/config/reference#unfocused-split-fill).

**Reply by Amparose** [reply]
Thanks! I didn't see those options.

I discovered that the problem shown in my screenshot is that, when in dark mode, if you refresh Ghostty's settings (⌘⇧,) it uses the light theme/s background colour as the overlay. Only way I've found to "fix" this is to restart Ghostty or cycle macOS's light/dark theme. I would assume this is a Ghostty bug.

**Reply by 00-kat** [reply]
I thought I remembered seeing something similar, but I can't find the discussion at the moment :(

**Comment by mitchellh** [comment, marked as answer]
#8282
